src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((str) => str.trim())
+    .filter((str) => str.length !== 0)
+    .reduce((prev, property) => {
+      const [name, value] = property.split(':');
+
+      prev[name.trim()] = value.trim();
+
+      return prev;
```

Comment:
According to the [CODE STYLE] checklist item, all chained methods called on the same level should have the same indentation to improve code readability. Make sure to align the methods accordingly, like in the provided good example.